src/logger.cpp

```diff
@@ -28,7 +28,7 @@ void Logger::logThread(intercept::types::r_string message) {
 }
 
 void Logger::pushTimestamp(std::ostream& str) const {
-      std::time_t t = std::time(0);   // get time now
+    std::time_t t = std::time(0);   // get time now
     std::tm* now = std::localtime(&t);
```

Comment:
actually, while copy pasting these a few days ago, both of these are actually unused, but doesn't matter

Reply:
I removed it then